compiler/src/iree/compiler/Dialect/HAL/IR/HALDialect.td

```diff
@@ -38,6 +38,7 @@ def HAL_Dialect : Dialect {
   }];
 
   let useDefaultAttributePrinterParser = 0;
+  let usePropertiesForAttributes = 0;
```

Comment:
I'm going to have a look at this when back at my desk and see if we can't fix it better.